src/lib/template-result.ts

```diff
@@ -28,7 +28,7 @@ export class TemplateResult {
 
   constructor(
       strings: TemplateStringsArray, values: any[], type: string,
-      processor: TemplateProcessor = defaultTemplateProcessor) {
+      processor: TemplateProcessor) {
```

Comment:
Note this breaking change in the CHANGELOG?

Reply:
Thanks! Done